### ISSUE TITLE
Skip to main content link is missing

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#680]
Det første elementet som kan brukes med tastaturet, skal være en lenke til hovedinnholdet på siden. Denne lenken skal være synlig når du fokuserer på den med tastaturet.

De fleste nettsider har en navigasjonsdel øverst med dusinvis av lenker (og muligens bilder, annonser og annet).

Seende besøkende kan bruke visuelle signaler til å omgå denne delen, men besøkende som bruker skjermlesere, kan bli tvunget til å høre på en lang liste over lenker, med mindre det finnes et alternativ for å hoppe over delen.

Lenken med teksten «Gå til hovedinnhold» eller «Hopp over navigering» skal være synlig i fokus slik at seende tastaturbrukere (for eksempel personer med motoriske funksjonshemninger) kan se at den finnes. Dermed kan de hoppe over lenker når de navigerer på siden.

Link til alle forekomster ligger i vedlagte Excel-ark:

[Issue#680 - Skip to main content link is missing.xlsx](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10686456/Issue.680.-.Skip.to.main.content.link.is.missing.xlsx)

[Skip to main content link is missing - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402469/Skip.to.main.content.link.is.missing.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)
